src/init/features/project.ts

```diff
@@ -32,48 +34,60 @@ export interface ProjectInfo {
  */
 export async function getProjectInfo(options: any): Promise<ProjectInfo> {
   if (options.project) {
-    return selectProjectFromOptions(options);
+    return getProjectFromId(options.project);
   }
-  return selectProjectFromList(options);
+  return selectProjectInteractively();
 }
 
 /**
  * Selects project when --project is passed in.
  * @param options Command line options.
  */
-async function selectProjectFromOptions(options: any): Promise<ProjectInfo> {
+async function getProjectFromId(projectId: string): Promise<ProjectInfo> {
   let project: FirebaseProjectMetadata;
   try {
-    project = await getFirebaseProject(options.project);
+    project = await getFirebaseProject(projectId);
   } catch (e) {
-    throw new FirebaseError(`Error getting project ${options.project}: ${e}`);
+    throw new FirebaseError(
+      `Error getting project ${projectId}. Please make sure the project exists and belongs to your account.`
```

Comment:
Update second sentence in message to `Please make sure the project exists and your account has permission to access it.`, since the user doesn't necessarily have to be the owner of the project to call getFirebaseProject.

Also, would it be useful to move this error message into getFirebaseProject so that other callers get the nice error message? And then you wouldn't have to try/catch here.

Reply:
Remove `getProjectFromId()` method since it can be replaced with a one-liner